(Doc+) Alerts UI cannot be CCS (#5513)
elastic/security-docs: docs/getting-started/data-views-in-sec.asciidoc

This only allows you to add index patterns that match indices that currently contain data (other index patterns are unavailable). Note that any changes made are saved in the current browser window and won't persist if you open a new tab.

> NOTE: You cannot update the data view for the Alerts page, e.g. to enable it to reference a cross-cluster search (CCS) data view. It always shows data from `.alerts-security.alerts-default`.

Reviewer: I think the CCS use case would be better as a separate sentence, so the first statement has more impact. Something like this:

```suggestion
NOTE: You cannot update the data view for the Alerts page. This includes referencing a cross-cluster search (CCS) data view or any other data view. The Alerts page always shows data from `.alerts-security.alerts-default`.
```
